e2e/integration/helpers/helpers.go

```diff
@@ -0,0 +1,341 @@
+	var temp map[string]interface{}
+
+	//containerName := "app-" + utils.GetRandomString(5)
+	Body := fmt.Sprintf(`{"apiVersion": "apps/v1","kind": "Deployment","metadata": {"name": "%s","labels": {"app": "nginx"}},
+				"spec": {"replicas": %v,"selector": {"matchLabels": {"app": "nginx"}},"template": {"metadata": {"labels": {"app": "nginx"}},
+				"spec": {"containers": [{"name": "nginx","image": "%s"}]}}}}`, deploymentName, replica, image)
+	err:= json.Unmarshal([]byte(Body), &temp)
+	if err!= nil{
+		utils.Failf("Unmarshal body failed: %v", err)
+		return err, temp
+	}
+
+	return nil, temp
+}
+
+//Function to handle app deployment/delete deployment.
+func HandleDeployment(operation string, apiserver string, UID string, ImageUrl string, replica int) bool {
+	var req *http.Request
+	var err error
+	var body io.Reader
+
+	client := &http.Client{}
+	switch operation {
+	case "POST":
+		Port := utils.RandomInt(10, 100)
+		err, body := GenerateDeploymentBody(UID, ImageUrl, Port, replica)
+		if err != nil {
+			utils.Failf("Payload marshalling failed: %v", err)
+		}
+		respbytes, err := json.Marshal(body)
+		if err != nil {
+			utils.Failf("Payload marshalling failed: %v", err)
+		}
+		req, err = http.NewRequest(http.MethodPost, apiserver, bytes.NewBuffer(respbytes))
+	case "DELETE":
+		req, err = http.NewRequest(http.MethodDelete, apiserver+UID, body)
+	}
+	if err != nil {
+		// handle error
+		utils.Failf("Frame HTTP request failed: %v", err)
+		return false
+	}
+	req.Header.Set("Content-Type", "application/json")
+	t := time.Now()
+	resp, err := client.Do(req)
+	utils.InfoV6("%s %s %v in %v", req.Method, req.URL, resp.Status, time.Now().Sub(t))
+	if err != nil {
+		// handle error
+		utils.Failf("HTTP request is failed :%v", err)
+		return false
+	}
+	return true
+}
+func DeleteDeployment(DeploymentApi, deploymentname string)int{
+	err, resp := SenHttpRequest(http.MethodDelete, DeploymentApi+"/"+deploymentname)
+	if err != nil {
+		// handle error
+		utils.Failf("HTTP request is failed :%v", err)
+		return -1
+	}
+
+	defer resp.Body.Close()
+
+	return resp.StatusCode
+
+}
+//Function to get the pods from Edged
+func GetPods(apiserver, label string) (v1.PodList, error) {
+	var pods v1.PodList
+	var resp *http.Response
+	var err error
+
+	ls := utils.MapLables(map[string]string{"app":label})
+	if len(ls) > 0{
+		err, resp = SenHttpRequest(http.MethodGet, apiserver+podLabelSelector+ls)
+	}else {
+		err, resp = SenHttpRequest(http.MethodGet, apiserver)
+	}
+	if err != nil {
+		utils.Failf("Frame HTTP request failed: %v", err)
+		return pods, nil
+	}
+	defer resp.Body.Close()
+	contents, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		utils.Failf("HTTP Response reading has failed: %v", err)
+		return pods, nil
+	}
+	err = json.Unmarshal(contents, &pods)
+	if err != nil {
+		utils.Failf("Unmarshal HTTP Response has failed: %v", err)
+		return pods, nil
+	}
+	return pods, nil
+}
+
+func GetPodState(apiserver string) (string, int){
```

Comment:
Pls add a comment with a short description above this function explaining its functionality